CLAUDE.md

```
@@ -0,0 +1,2414 @@
+```redis
+SHIELD.stats ML
+→ 1) model_version: "v1.2.3"
+   2) predictions_count: 1000000
+   3) avg_inference_latency_ms: 0.8
+   4) anomaly_detection_rate: 0.02
+   5) false_positive_rate: 0.001
+   6) model_accuracy: 0.94
+```
+
+**A/B Testing:**
+```redis
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)